May Serenity updates (#598)
akeneo/pim-helpcenter: content/updates/2020-05/1-new-connection-monitoring-page.md

# Connections new features

## Easily monitor errors on your connections 

> ::: meta-data type="New" features="Connections, Productivity" available="mid-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"

Reviewer: ```suggestion
::: meta-data type="New" features="Connections, Productivity, Performance" available="mid-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
```

:::

> The new `Monitoring` page available under `Connections` lists the last errors for your Connections. From this page, you can now easily track any error on products creation, update, or deletion for each of your connections and fix it. 

Reviewer 2: ```suggestion
The new `Monitoring` page available under `Connections` lists the last errors for your Connections. From this page, you can now easily track and fix any error on product creation, update, or deletion for each of your connections. 
```

![Connection monitoring page](../img/new-connection-monitoring-page.png)

## New metrics on the Connection dashboard
::: meta-data type="New" features="Connections, Productivity" available="late-May" link-to-doc="../articles/connection-dashboard.html#error-monitoring"
:::

> To ease error monitoring and allow you to see at a glance how your Source Connections are performing, the connection dashboard now displays: 

Reviewer: ```suggestion
To ease error monitoring and allow you to see at a glance how your Source Connections are performing, the Connection dashboard now displays: 
```
Capital C?

- a graph with the number of product errors (business and technical) for the past week,

> - and connection cards showing the number of business errors for the past week. 

Reviewer: ```suggestion
- connection cards showing the number of business errors for the past week. 
```
